<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCheckDialog</class>
 <widget class="QDialog" name="AddCheckDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Кассир: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Фотограф: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Стоимость:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="OutDE">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>85</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Время:</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="OutTE">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>85</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="WorksTableView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>571</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddWorkBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить работу</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Работы:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EditWorkBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>460</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить работу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveWorkBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>460</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить работу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AcceptCheck">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>460</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Подтвердить</string>
   </property>
  </widget>
  <widget class="QLabel" name="CashierNameLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ФИО</string>
   </property>
  </widget>
  <widget class="QLabel" name="CostLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QComboBox" name="PhotographerNameCB">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>55</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="StatusNameLbl">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Создание</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Статус:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="NowBtn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>85</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Сейчас</string>
   </property>
  </widget>
  <widget class="QLabel" name="PhotographerNameLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ФИО</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
